### ISSUE TITLE
Docs: Error logging example

### ISSUE THREAD

**Issue by walles** [lightstep/lightstep-census-java#14]
Hi!

I want to tag a span as failing, with a stack trace and an exception class, as described here:
<https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table>

The best way I have come up with so far for the logging part is to use an annotation:
```java
    Tracing.getTracer()
        .getCurrentSpan()
        .addAnnotation(
            Annotation.fromDescriptionAndAttributes(
                "I don't want this string in the Lightstep UI",
                Collections.singletonMap("error.kind", AttributeValue.stringAttributeValue(e.getClass().getSimpleName()))));
```

It works, but it also clutters the Lightstep UI with the `I don't want this string in the Lightstep UI` string.

What is the recommended way to do this? How do I do it if I don't want the `I don't want this string in the Lightstep UI` string in the Lightstep UI?

Related to #12; both issues are about documenting how to use this package.

**Comment by walles** [comment]
Also, I just realized that `error.object` should be...
> the actual Throwable/Exception/Error object instance itself. E.g., A `java.lang.UnsupportedOperationException` instance

... according to this documentation: <https://github.com/opentracing/specification/blob/master/semantic_conventions.md#log-fields-table>

What would be the easiest way for me to provide that using `lightstep-census-java`?

**Comment by walles** [comment]
For reference, here's what I'm doing now, it kind of works but is not great.

If you could provide a better function that...
* Does not get log any description
* Logs the error object itself rather than its `String`ified backtrace
... that would be super!

Also, the stack trace lacks linebreaks in the Lightstep UI, making it hard to follow, not sure if that's something I can do anything about on my end:
<img width="472" alt="stacktrace-without-linebreaks" src="https://user-images.githubusercontent.com/158201/76412694-545e5780-6394-11ea-8bfc-dbd3455e6ddd.png">

```java
  public static void logException(Throwable logMe) {
    Map<String, AttributeValue> logs = new HashMap<>();

    logs.put("error.kind", AttributeValue.stringAttributeValue(logMe.getClass().getSimpleName()));

    StringWriter stringWriter = new StringWriter();
    PrintWriter printWriter = new PrintWriter(stringWriter);
    logMe.printStackTrace(printWriter);
    // FIXME: How to log the logMe object itself, rather than its stack trace string?
    logs.put("error.object", AttributeValue.stringAttributeValue(stringWriter.toString()));

    // FIXME: How to get rid of this?
    final String unwantedDescription = "Logged using SpTracing.logException()";

    Tracing.getTracer()
        .getCurrentSpan()
        .addAnnotation(Annotation.fromDescriptionAndAttributes(unwantedDescription, logs));
  }
```

**Comment by malafeev** [comment]
I don't see how properly log exception using OpenCensus span.
From [documentation](https://opencensus.io/quickstart/java/tracing/#using-the-tracer):
```
span.setStatus(Status.INTERNAL.withDescription(e.toString()));
```
Maybe it makes sense to ask OpenCensus developers.
